src/util/transform_matrix_removal.ts

```diff
@@ -0,0 +1,60 @@
+import { Image } from '../shapes/image.class';
+import type { FabricObject } from '../shapes/Object/FabricObject';
+import { TMat2D } from '../typedefs';
+import { qrDecompose } from './misc/matrix';
+
+type FabricObjectWithTransformMatrix = FabricObject & {
+  transformMatrix?: TMat2D;
+};
+
+/**
+ * This function is an helper for svg import. it decompose the transformMatrix
+ * and assign properties to object.
+ * untransformed coordinates
+ * @private
+ */
+export const _assignTransformMatrixProps = (
+  object: FabricObjectWithTransformMatrix
+) => {
+  if (object.transformMatrix) {
+    const { scaleX, scaleY, angle, skewX } = qrDecompose(
+      object.transformMatrix
+    );
+    object.flipX = false;
+    object.flipY = false;
+    object.set('scaleX', scaleX);
+    object.set('scaleY', scaleY);
+    object.angle = angle;
+    object.skewX = skewX;
+    object.skewY = 0;
+  }
+};
+
+/**
+ * This function is an helper for svg import. it removes the transform matrix
+ * and set to object properties that fabricjs can handle
+ * @private
+ * @param {Object} preserveAspectRatioOptions
+ */
+export const _removeTransformMatrix = (
```

Comment:
naming?

Reply:
don't know pick one. I think exportin only one is good enough